src/NServiceBus.Persistence.AzureStorage.ComponentTests/NServiceBus.Persistence.AzureStorage.ComponentTests.csproj

```diff
@@ -2,30 +2,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>PCR0001</NoWarn>
-    <RootNamespace>NServiceBus.Persistence.AzureStorage.ComponentTests</RootNamespace>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Is this necessary?

Reply:
allows us to remove the binding redirects

Reply:
From app.config, but do we really need the redirects in place?